docs/ops/config.rst

```diff
@@ -0,0 +1,91 @@
+Configuration
+=============
+All configuration happens through environment variables.
+
+Django settings
+---------------
+These settings map directly to built-in Django settings. An environment
+variable like ``DJANGO_FOO`` controls the Django setting ``FOO``. Not all
+Django settings are available for configuration.
+
+``DATABASE_URL``
+    A `database url`_, including any username and password, if needed.
+    Default ``postgres://postgres@localhost/normandy``.
+``DJANGO_SECRET_KEY``
+    A string used to as a seed for security features. This should be the same
+    on all instances that share a database, and should be kept secret. It
+    should be a long, random string. Required, no default.
+``DJANGO_DEBUG``
+    ``true`` or ``false``. Default ``false``. Enables Django's debug mode. This
+    should never be enabled on permanent servers. It is inefficient and leaks
+    memory.
+``DJANGO_MEDIA_URL``
+    The url to prefix for media files (files uploaded to the service).
+    Default ``/media/``. Both host-relative (``/media/``) and host-absolute
+    urls (``https://cdn.example.com/``) work. Should end in a slash.
+``DJANGO_STATIC_URL``
+    The url to prefix for static files (files shipped with the service).
+    Default ``/static/``. Both host-relative and host-absolute urls work.
+    Should end in a slash.
+``DJANGO_CONFIGURATION``
+    The name of the set of configurations to use as the base. Default
```

Comment:
This wording is a bit weird. Maybe "The name of a configuration preset to use for this environment."